lesson17/index.html

```diff
@@ -5,15 +5,12 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>lesson 16</title>
+  <title>lesson 17</title>
   <link rel="stylesheet" href="style.css">
 </head>
 
 <body id="js-body">
   <main>
-    <div class="news" id="news">
-      <ul class="news_Tabs" id="news_Tabs"></ul>
-    </div>
   </main>
   <script src="./index.js"></script>
```

Comment:
IMO
To the 8th line.　
```suggestion
 
```

Reply:
Thank you!
I fixed it in my next comment.